bin/git-plop

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+find_plop_json() {
+  current_dir=$(pwd)
+
+  while [ "$current_dir" != "/" ]; do
+    if [ -f "$current_dir/.plop.json" ]; then
+      echo "$current_dir/.plop.json"
+      return 0
+    fi
+    current_dir=$(dirname "$current_dir")
+  done
+
+  echo "Error: .plop.json not found"
+  return 1
+}
+
+plop_json_path=$(find_plop_json)
+
+if [ "$plop_json_path" == "Error: .plop.json not found" ]; then
+  echo "$plop_json_path"
+  exit 1
+fi
+
+plop_json_dir=$(dirname "$plop_json_path")
+
+src=$(jq -r '.src' "$plop_json_path")
+workdir=$(jq -r '.workdir' "$plop_json_path")
+
+src_dir="$plop_json_dir/$src"
+workdir_dir="$plop_json_dir/$workdir"
+
+echo "Source directory: $src_dir"
+echo "Working directory: $workdir_dir"
+
+if [ ! -f "$plop_json_dir/package.json" ]; then
+  echo "Error: package.json not found in $plop_json_dir"
+  exit 1
+fi
+
+docker run -it --rm -v "$plop_json_dir:/usr/src/app" -v "$src_dir:/usr/src/src" -v "$workdir_dir:/usr/src/modules" plop_plop:latest bash -c "npm run plop"
```

Comment:
wrong mount directories.
you need to mount:

src dir > /plop
workdir > /codebase

---

I think will be nice to have: also provide env variable about workdir, so we don't hard-code /codebase in our plop file and just take NODE_ENV. to do that add to docker run:
`-e CODEBASE_DIR=$workdir_dir`

and we will have NODE_ENV.CODEBASE_DIR